runtime/shared_common/CacheLifecycleManager.cpp

```diff
@@ -286,9 +285,10 @@ printSharedCache(void* element, void* param)
 			j9tty_printf(PORTLIB, "\nIncompatible shared caches\n");
 			state->printIncompatibleHeader = 2;
 		}
-
 		j9tty_printf(PORTLIB, "%-16s\t", currentItem->name);
-		getStringForShcModlevel(PORTLIB, currentItem->versionData.modlevel, (char*)&jclLevelStr);
+		char jclLevelStr[10];
+		memset(jclLevelStr, 0, sizeof(jclLevelStr));
+		getStringForShcModlevel(PORTLIB, currentItem->versionData.modlevel, (char*)&jclLevelStr, sizeof(jclLevelStr));
```

Comment:
I'm surprise no compilers have complained about taking the address of an array. The name of an array yields (an r-value) which is the address of the first element of that array - you can't (normally) take the address of an r-value.

Instead of `(char*)&jclLevelStr` it should be just `jclLevelStr`. @hangshao0 Can you put together a new pull request that fixes that sort of problem in at least this file? Thanks.

Reply:
New PR here: https://github.com/eclipse/openj9/pull/1719